src/commands/acronym.js

```diff
@@ -17,22 +17,25 @@ module.exports = {
   */
   async execute(message, args) {
     if (this.args && args.length === 0) {
-      try {
-        message.reply(`*Give me more data* para eu poder trabalhar!\n${this.usage}`);
-      } catch (e) {
-        //
-      }
+      message.reply(`*Give me more data* para eu poder trabalhar!\n${this.usage}`);
```

Comment:
Better have it all in portuguese?

Reply:
Was already like that but agree with you.

@JorgeMiguelGomes should we update the message to be all in portuguese?